Diagnostics/HealthChecker/DataCollection/ExchangeInformation/Get-ExchangeAMSIConfigurationState.ps1

```diff
@@ -0,0 +1,57 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+. $PSScriptRoot\..\..\Helpers\Invoke-CatchActions.ps1
+Function Get-ExchangeAMSIConfigurationState {
+
+    begin {
+        Write-Verbose "Calling: $($MyInvocation.MyCommand)"
+    } process {
+        try {
+            Write-Verbose "Trying to query AMSI configuration state"
+            $amsiConfiguration = Get-SettingOverride | Where-Object { ($_.ComponentName -eq "Cafe") -and ($_.SectionName -eq "HttpRequestFiltering") } -ErrorAction Stop
```

Comment:
Need to move the `-ErrorAction Stop` to before the pipe and have it be with `Get-SettingOverride`

Reply:
Done.